.ci/azure/linux_arm64.yml

```diff
@@ -63,66 +104,235 @@ jobs:
-      # OpenCV should provide install_build_dependencies.sh as well
-      # Move into resources
-      git clone https://github.com/opencv/opencv.git --depth 1 $(OPENCV_REPO_DIR)
+      sudo apt --assume-yes install scons crossbuild-essential-arm64 libprotoc-dev libhiredis-dev
+      sudo apt --assume-yes install protobuf-compiler default-jdk libssl-dev zip libzstd-dev python-dev
       # Speed up build
       wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
       unzip ninja-linux.zip
       sudo cp -v ninja /usr/local/bin/
+      # OpenCV should provide install_build_dependencies.sh as well
+      # Move into resources
+      git clone https://github.com/opencv/opencv.git --depth 1 $(REPO_OPENCV)
     workingDirectory: $(WORK_DIR)
     displayName: 'Install dependencies'
 
+  - script: |
+      #### Build python for host 
+      wget https://www.python.org/ftp/python/$(PYTHON_ARM_VERSION)/Python-$(PYTHON_ARM_VERSION).tgz
+      tar -xzf Python-$(PYTHON_ARM_VERSION).tgz
+      mv Python-$(PYTHON_ARM_VERSION) Python-$(PYTHON_ARM_VERSION)-host
+      cd Python-$(PYTHON_ARM_VERSION)-host
+      ./configure --enable-optimizations
+      make -j $(NUM_PROC) python Parser/pgen
+      sudo make -j $(NUM_PROC) install
+      cp python $(BUILD_PYTHON)
+      cp -r Parser/pgen $(BUILD_PYTHON)
+      curl https://bootstrap.pypa.io/get-pip.py | $(BUILD_PYTHON)/python - --no-cache-dir numpy cython
+      cd ..
+      #### Build python for ARM 
+      wget https://www.python.org/ftp/python/$(PYTHON_ARM_VERSION)/Python-$(PYTHON_ARM_VERSION).tgz
+      tar -xzf Python-$(PYTHON_ARM_VERSION).tgz
+      cd Python-$(PYTHON_ARM_VERSION)
+      CC=aarch64-linux-gnu-gcc \
+      CXX=aarch64-linux-gnu-g++ \
+      AR=aarch64-linux-gnu-ar \
+      READELF=aarch64-linux-gnu-readelf \
+      RANLIB=aarch64-linux-gnu-ranlib \
+          ./configure \
+          --build=x86_64-linux-gnu \
+          --host=aarch64-linux-gnu \
+          --prefix=$(INSTALL_PYTHON) \
+          --disable-ipv6 ac_cv_file__dev_ptmx=no ac_cv_file__dev_ptc=no ac_cv_have_long_long_format=yes \
+          --enable-shared
+      make -j $(NUM_PROC) \
+          HOSTPYTHON=$(BUILD_PYTHON)/python \
+          HOSTPGEN=$(BUILD_PYTHON)/Parser/pgen \
+          CROSS-COMPILE=aarch64-linux-gnu- CROSS_COMPILE_TARGET=yes HOSTARCH=aarch64-linux BUILDARCH=aarch64-linux-gnu
+      make -j $(NUM_PROC) install
+      cd ..
+      sudo /usr/local/bin/python3.8 -m pip install numpy cython 
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Build python for ARM'
+
   - task: CMake@1
     inputs:
       cmakeArgs: >
-        -GNinja
-        -DVERBOSE_BUILD=ON
-        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
-        -DBUILD_LIST=imgcodecs,videoio,highgui
-        -DCMAKE_TOOLCHAIN_FILE=$(OPENCV_REPO_DIR)/platforms/linux/aarch64-gnu.toolchain.cmake
-        $(OPENCV_REPO_DIR)
-      workingDirectory: $(BUILD_DIR_OPENCV)
+        -G Ninja 
```

Comment:
it will be good to add a script for building openCV too (or into contrib install_build_dependencies.sh)